docs/faq.md

```diff
@@ -560,3 +560,22 @@ In `/var/www/html/Lychee/.env`, change `DB_HOST=localhost` to `DB_HOST=127.0.0.1
 Lychee checks for the presence of certain files to detect whether Lychee was installed as a git repository or from a release archive (.zip). It also uses them to determine which git commit is in use where applicable. This is included in the Diagnostics page to assist us in diagnosing issues.
 
 These messages can be safely ignored.
+
+### Import from server via symlink is failing.
+
+If you check the logs and see an output like this:
+
+```
+2021-11-09 19:09:50 -- error -- App\Actions\Import\Exec::do -- 256 -- Could not import file (<path-to-image>/image.jpg): 0 symlink(): Operation not supported
+```
+
+This means the native low-level function `symlink` fails, because the file system does not support symbolic links.
+
+When using this import setting, symbolic links are created below the `/uploads` directory as this is the folder where the media files reside.
+In practice, either the media files are copied into this directory (if truly imported without symbolic links),
+or symbolic links are created when _import with symbolic links_ is activated.
```

Comment:
Why is this sentence here? IMHO it interrupts the train of thoughts. In particular as the next sentence start with "As a result" but this refers to the previous sentence which says "symbolic links are created". I don't think that is sentence serves any purpose.

If for what ever reasons why need to keep this sentence, why does it start with "In practice"?

Reply:
True. It's redundant as the question is only about symlinks.

Reply:
Fixed by https://github.com/LycheeOrg/LycheeOrg.github.io/pull/61/commits/dda94b818edb967874627448b7a77928a00be611